source/containers.tex

```diff
@@ -861,7 +861,8 @@
 \rSec3[container.alloc.reqmts]{Allocator-aware containers}
 
 \pnum
-All of the containers defined in \ref{containers} and in~\ref{basic.string} except \tcode{array}
+All of the containers defined in \ref{containers} except \tcode{array},
+in \ref{stacktrace.basic}, in \ref{basic.string}, and in \ref{re.results}
```

Comment:
No change requested, but something to consider: IMO this would read better if the "in containers except array" case was last, but it would seem quite odd not to list the "real" containers first.

Reply:
I considered this, but to avoid all confusion I wanted "in \ref{containers} except \tcode{array}" to clearly be a single item in the list, where placing this last requires attention to the Oxford comma to understand the intent.  The difference has no impact on interpretation, as `array` *is specified in* the [containers] clause, but I feel it gives an easier parse so the reader never worries about those details.  So yes, this ordering was intentional :) but I am open to others' view on clarity/ease-of-reading.

Reply:
Maybe put the exception first:

"Except for array, all of the containers defined in a,b,c meet the additional requirements..."

This doesn't tell you where "array" is defined, but maybe that's not super-important here.

Reply:
Thank's, Jen's!